locale/coa/us/General.xml

```diff
@@ -159,6 +160,7 @@
   </currencies>
   <settings>
     <setting name="inventory_accno_id" accno="1510"/>
+    <setting name="earn_id" accno="3500"/>
```

Comment:
This is the reason the tests fail: setting the `earn_id` indicates to the system that it has been set up for hierarchy reporting and changes the default setup of the balance sheet and PNL from "account category basis" to "hierarchy". The two result in different roll-up of PNL results into "current earnings".

Reply:
Ok, so I'll remove earn_id, but it still does not seem right that Income goes to Freight. So Should I make a new pull request with just that?

Reply:
Removing this setting from the PR (we can always squash the commits) and then merging the PR does make sense, I'd say, yes.